server/services/import/import-v2.js

```diff
@@ -157,22 +250,44 @@ const updateOrCreate = async (user, slug, datum, idField = 'id', { importStage }
   datum = cloneDeep(datum);
 
   if (importStage == 'simpleAttributes') {
-    const attributeNames = getModelAttributes(slug, { filterOutType: ['component', 'dynamiczone', 'media', 'relation'], addIdAttribute: true })
+    const attributeNames = getModelAttributes(slug, {
```

Comment:
These changes should not appear since it's just linting. Configure your linter properly to rollback changes